### ISSUE TITLE
Replication of comparable glow with papers

### ISSUE THREAD

**Issue by prclibo** [VincentStimper/normalizing-flows#5]
Hi, Thanks for developing this package. I find it very neat and flexible and would like to use it for my research. I noticed that in the paper "Resampling Base Distributions of Normalizing Flows", the bpd of your glow can reach 3.2~3.3, which is comparable to the original paper.

I was wondering if it is possible to share your training scripts and details to train glow on cifar10 to achieve the above bpd. The current example notebook is too sketchy and only results in bpd of 3.8. Thanks very much!

**Comment by VincentStimper** [comment]
Hi @prclibo,
for the paper [Resampling Base Distributions of Normalizing Flows](https://arxiv.org/abs/2110.15828) I created a [separate repository](https://github.com/VincentStimper/resampled-base-flows). You can use the script [`train_glow.py`](https://github.com/VincentStimper/resampled-base-flows/blob/master/experiments/train_glow.py) together with the configuration file [`gauss_32.yaml`](https://github.com/VincentStimper/resampled-base-flows/blob/master/experiments/config/cifar10/gauss_32.yaml) if you want to use a Gaussian base distribution or [`resampled_32.yaml`](https://github.com/VincentStimper/resampled-base-flows/blob/master/experiments/config/cifar10/resampled_32.yaml) if you want to use the resampled base distribution we introduced in the paper. You should get bpd values of less then 3.3 but notice that depending on the hardware you use this might take several days.
Let me if you still have problems reproducing the results.
